catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/FrameworkProperties.java

```diff
@@ -80,7 +80,7 @@
 
   private List<ConnectedSource> connectedSources = new ArrayList<>();
 
-  private Collection<FederatedSource> federatedSources = Collections.emptyList();
+  private List<FederatedSource> federatedSources = Collections.emptyList();
```

Comment:
There was a strange issue in iTests when registering and federating with custom sources that seemed to be resolved when swapping this to a list.